sbi/inference/snpe/snpe_base.py

```diff
@@ -369,17 +369,9 @@ def _train(
                     batch[2].to(self._device),
                 )
 
-                # Just do maximum likelihood in the first round.
-                if round_ == 0:
-                    log_prob = self._neural_posterior.neural_net.log_prob(
-                        theta_batch, x_batch
-                    )
-                else:  # or call the SnpeC loss
-                    log_prob = self._get_log_prob_proposal_posterior(
-                        theta_batch, x_batch, masks
-                    )
-                loss = -torch.mean(log_prob)
-                loss.backward()
+                # Take mean loss over batch.
```

Comment:
I think this comment is superfluous, unless you explain that the batch is of scalars (log probs) so the mean doesn't need a `dim`.